app/Services/Phoenix/Phoenix.php

```diff
@@ -48,4 +48,22 @@ public function getCampaign($id)
 
         return is_null($response) ? null : $response;
     }
+
+    /**
+     * Send a GET request to return a reportback & an item with the specified id.
+     *
+     * @param  string $reportback_id
+     * @param  string $reportback_item_id
+     * @return object|null
+     */
+    public function getSpecificReportbackItem($reportback_id, $reportback_item_id)
+    {
+        $response = $this->get($this->base_uri . 'reportbacks/' . $reportback_id);
```

Comment:
I don't think we need to pass `$this->base_uri .` here. This gets set in the `__construct` for the `Phoenix` class... but it did highlight that in the `getCampaigns()` we're doing that too. We should be able to remove in both and simplify the call to just:

```php
$response = $this->get('reportbacks/' . $reportback_id);
```